### DISCUSSION TITLE
how to get the full path of the process executable file？

### DISCUSSION THREAD

**Discussion by qiaoms-cloud** [cilium/ebpf#984] [Q&A] [answered]
I can not get  the full path of the process executable file from the struct task，who can help me?

**Comment by lmb** [comment, marked as answer]
Hi, this isn't the right venue for general BPF questions. Please try [StackOverflow](https://stackoverflow.com/questions/tagged/ebpf+or+bpf+or+xdp-bpf) or the [slack channel](https://cilium.herokuapp.com/).
